puppet_litmus.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
     Providing a simple command line tool for puppet content creators, to enable simple and complex test deployments.
   EOF
   spec.summary = 'Providing a simple command line tool for puppet content creators, to enable simple and complex test deployments.'
-  spec.add_runtime_dependency 'bolt',        ['>= 1.13.1', '< 2.0.0']
+  spec.add_runtime_dependency 'bolt',        ['>= 2.0.0', '< 3.0.0']
```

Comment:
Can we bump the lower bound to 2.0.1 so we have the inventory hash fix from https://github.com/puppetlabs/bolt/commit/345e5172fc6f98055e87911b157551a6cafae32d

Reply:
addressed. I've also removed the extraneous clones. But see https://github.com/puppetlabs/puppet_litmus/pull/254/commits/21dbbc749f7ea661bebab5482a7538eca0cf8ad1 for more fun.